CourseGrab/GoogleService-Info.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>894460636145-mulqctlklp1alf73u43kbdm5n7vulqga.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.894460636145-mulqctlklp1alf73u43kbdm5n7vulqga</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>org.cuappdev.CourseGrab</string>
+</dict>
+</plist>
```

Comment:
this file should also be in the `.gitignore` and we should not be committing this